pkg/actors/actor.go

```diff
@@ -50,14 +50,14 @@ func (a *actor) lock() {
 	a.concurrencyLock.Lock()
 
 	a.busy = true
-	a.busyCh = make(chan bool, 1)
 	a.lastUsedTime = time.Now().UTC()
 }
 
 func (a *actor) unLock() {
 	if a.busy {
 		a.busy = false
-		close(a.busyCh)
+		// this signals to drain actors earlier than DrainOngoingCallTimeout.
+		a.busyCh <- false
```

Comment:
@yaron2 please double-check that this change will not impact the current draining behavior. 

this is covered by the below existing test.

https://github.com/dapr/dapr/blob/9e87443bb625674489e046fe3eb96ebc9d86f74b/pkg/actors/actor_test.go#L52-L71